### ISSUE TITLE
MultiSelect: Screen reader does not read the selected value

### ISSUE THREAD

**Issue by veekeys** [iTwin/iTwinUI-react#794]
https://github.com/iTwin/iTwinUI-react/pull/700#discussion_r951662931
